--- a/browser/base/content/browser-menubar.inc
+++ b/browser/base/content/browser-menubar.inc
@@ -432,6 +432,7 @@
                           disabled="true"
                           appmenu-data-l10n-id="appmenuitem-report-broken-site"/>
                 <menuitem id="feedbackPage"
+                          hidden="true"
                           data-l10n-id="menu-help-share-ideas"
                           appmenu-data-l10n-id="appmenu-help-share-ideas"/>
                 <menuitem id="helpSafeMode"
